mix.exs

```diff
@@ -55,7 +55,7 @@ defmodule Membrane.MP4.Plugin.MixProject do
       {:membrane_cmaf_format, "~> 0.1.0"},
       {:membrane_aac_format, "~> 0.1.0"},
       {:membrane_http_adaptive_stream_plugin, "~> 0.1.0"},
-      {:membrane_caps_video_h264, "~> 0.1.0"},
+      {:membrane_caps_video_h264, "~> 0.2.0"},
```

Comment:
that was an opportunity to rename this package :|

Reply:
it's used by membrane_element_ffmpeg and I-don't-know-what-else, so this would involve updating (and possibly renaming) more stuff

Reply:
and there's always a better day for this than today :D